aperturedb/ParallelLoader.py

```diff
@@ -118,19 +119,24 @@ def print_stats(self):
         print("============ ApertureDB Loader Stats ============")
         print("Total time (s):", self.ingestion_time)
         print("Total queries executed:", total_queries_exec)
-        print("Avg Query time (s):", np.mean(times))
-        print("Query time std:", np.std (times))
-        print("Avg Query Throughput (q/s)):",
-                1 / np.mean(times) * self.numthreads)
 
-        msg = "(" + self.type + "/s):"
-        print("Overall insertion throughput", msg,
-                self.total_elements / self.ingestion_time)
+        if total_queries_exec == 0:
```

Comment:
is this to prevent division by 0 on l139? It might still be of use to have some times if they get reported, though I have not run this enough times to see how those counters fill up.

Reply:
yes, it is mainly to prevent division by zero. When no queries run, it is because there is something off with the input data (an exception is usually thrown). There is no need to report any other metric if no query succeeded.